`make tryout` on fresh Git clone fails with error

OS: Void Linux x86_64 (glibc)
Emacs version (from `emacs --version`): `GNU Emacs 25.1.1`

When I do a fresh clone of this project and run `make tryout`, the command fails with the error below:
```sh
$ cd $(mktemp -d)
$ git clone --depth 1 git://github.com/tkf/emacs-jedi.git
Cloning into 'emacs-jedi'...
remote: Counting objects: 48, done.
remote: Compressing objects: 100% (36/36), done.
remote: Total 48 (delta 3), reused 24 (delta 1), pack-reused 0
Receiving objects: 100% (48/48), 48.53 KiB | 0 bytes/s, done.
Resolving deltas: 100% (3/3), done.
$ cd emacs-jedi
$ make tryout
cask install
test -d /tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa
touch /tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa
rm -rf *.elc
cask exec emacs -Q --eval "(setq python-environment--verbose t)" --eval "(setq jedi:environment-root \"/tmp/tmp.SGG7Df9YLy/emacs-jedi/env\")" -batch \
	-L . -f batch-byte-compile *.el

In toplevel form:
jedi-core.el:495:1:Warning: Unused lexical variable ‘mngr-list’
jedi-core.el:649:1:Warning: Unused lexical argument ‘x’
jedi-core.el:826:1:Warning: Unused lexical argument ‘x’
jedi-core.el:980:1:Warning: Unused lexical argument ‘x’

In jedi:dot-complete:
jedi.el:93:16:Warning: looking-back called with 1 argument, but requires 2-3

In toplevel form:
test-jedi.el:64:25:Warning: assignment to free variable
    ‘python-indent-guess-indent-offset’
test-jedi.el:64:25:Warning: ‘font-lock-fontify-buffer’ is for interactive use
    only; use ‘font-lock-ensure’ or ‘font-lock-flush’ instead.
test-jedi.el:68:14:Warning: ‘font-lock-fontify-buffer’ is for interactive use
    only; use ‘font-lock-ensure’ or ‘font-lock-flush’ instead.
test-jedi.el:80:14:Warning: ‘font-lock-fontify-buffer’ is for interactive use
    only; use ‘font-lock-ensure’ or ‘font-lock-flush’ instead.
test-jedi.el:94:14:Warning: ‘font-lock-fontify-buffer’ is for interactive use
    only; use ‘font-lock-ensure’ or ‘font-lock-flush’ instead.
test-jedi.el:276:14:Warning: ‘font-lock-fontify-buffer’ is for interactive use
    only; use ‘font-lock-ensure’ or ‘font-lock-flush’ instead.
cask exec emacs -Q --eval "(setq python-environment--verbose t)" --eval "(setq jedi:environment-root \"/tmp/tmp.SGG7Df9YLy/emacs-jedi/env\")" -batch -L . -l jedi.el -f "jedi:install-server-block"
Making virtualenv at /tmp/tmp.SGG7Df9YLy/emacs-jedi/env (SYNC)...
New python executable in /tmp/tmp.SGG7Df9YLy/emacs-jedi/env/bin/python2
Also creating executable in /tmp/tmp.SGG7Df9YLy/emacs-jedi/env/bin/python
Installing setuptools, pip, wheel...done.

Making virtualenv at /tmp/tmp.SGG7Df9YLy/emacs-jedi/env (SYNC)...Done
Running: pip install --upgrade /tmp/tmp.SGG7Df9YLy/emacs-jedi/ (SYNC)...
Processing /tmp/tmp.SGG7Df9YLy/emacs-jedi
Exception:
Traceback (most recent call last):
  File "/tmp/tmp.SGG7Df9YLy/emacs-jedi/env/lib/python2.7/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/tmp/tmp.SGG7Df9YLy/emacs-jedi/env/lib/python2.7/site-packages/pip/commands/install.py", line 335, in run
    wb.build(autobuilding=True)
  File "/tmp/tmp.SGG7Df9YLy/emacs-jedi/env/lib/python2.7/site-packages/pip/wheel.py", line 749, in build
    self.requirement_set.prepare_files(self.finder)
  File "/tmp/tmp.SGG7Df9YLy/emacs-jedi/env/lib/python2.7/site-packages/pip/req/req_set.py", line 380, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/tmp/tmp.SGG7Df9YLy/emacs-jedi/env/lib/python2.7/site-packages/pip/req/req_set.py", line 620, in _prepare_file
    session=self.session, hashes=hashes)
  File "/tmp/tmp.SGG7Df9YLy/emacs-jedi/env/lib/python2.7/site-packages/pip/download.py", line 809, in unpack_url
    unpack_file_url(link, location, download_dir, hashes=hashes)
  File "/tmp/tmp.SGG7Df9YLy/emacs-jedi/env/lib/python2.7/site-packages/pip/download.py", line 686, in unpack_file_url
    shutil.copytree(link_path, location, symlinks=True)
  File "/usr/lib64/python2.7/shutil.py", line 208, in copytree
    raise Error, errors
Error: [(’/tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent.ssh’, ’/tmp/pip-0fc1yc-build/.cask/25.1/elpa/gnupg/S.gpg-agent.ssh’, "[Errno 6] No such device or address: ’/tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent.ssh’"), (’/tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent.browser’, ’/tmp/pip-0fc1yc-build/.cask/25.1/elpa/gnupg/S.gpg-agent.browser’, "[Errno 6] No such device or address: ’/tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent.browser’"), (’/tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent.extra’, ’/tmp/pip-0fc1yc-build/.cask/25.1/elpa/gnupg/S.gpg-agent.extra’, "[Errno 6] No such device or address: ’/tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent.extra’"), (’/tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent’, ’/tmp/pip-0fc1yc-build/.cask/25.1/elpa/gnupg/S.gpg-agent’, "[Errno 6] No such device or address: ’/tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent’")]

Running: pip install --upgrade /tmp/tmp.SGG7Df9YLy/emacs-jedi/ (SYNC)...Done
Command ("/tmp/tmp.SGG7Df9YLy/emacs-jedi/env/bin/pip" "install" "--upgrade" "/tmp/tmp.SGG7Df9YLy/emacs-jedi/") exits with error code 2.
make: *** [Makefile:76: env/bin/jediepcserver] Error 255

```

I believe the error to be related to a `gpg-agent` process which is somehow started by (and then left running after) `make tryout`:
```sh
$ ps ax|grep "gpg-agent.*emacs-jedi"
  949 ?        Ss     0:00 gpg-agent --homedir /tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg --use-standard-socket --daemon
 1990 pts/5    S+     0:00 grep gpg-agent.*emacs-jedi
```
This `gpg-agent` is using the (socket) files mentioned in the error message:
```sh
$ lsof -p 949 | grep emacs-jedi
gpg-agent 949 miki    3u     unix 0x0000000000000000      0t0  201287 /tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent type=STREAM
gpg-agent 949 miki    4u     unix 0x0000000000000000      0t0  201289 /tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent.extra type=STREAM
gpg-agent 949 miki    5u     unix 0x0000000000000000      0t0  201291 /tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent.browser type=STREAM
gpg-agent 949 miki    6u     unix 0x0000000000000000      0t0  201293 /tmp/tmp.SGG7Df9YLy/emacs-jedi/.cask/25.1/elpa/gnupg/S.gpg-agent.ssh type=STREAM
```

--- a/Cask
+++ b/Cask
@@ -3,9 +3,10 @@
 
 (package "jedi" "0" "A Python auto-completion for Emacs")
 
+(depends-on "epc")
+(depends-on "python-environment")
+
 (development
  (depends-on "ert")
- (depends-on "epc")
  (depends-on "auto-complete")
- (depends-on "python-environment")
  (depends-on "mocker"))

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-ENV = env
+ENV = $(PWD)/env
 
 VIRTUALENV_SYSTEM_SITE_PACKAGES ?= true
 VIRTUALENV = \
@@ -19,7 +19,7 @@ ELPA_DIR = $(shell EMACS=$(EMACS) $(CASK) package-directory)
 
 VIRTUAL_EMACS = ${CASK} exec ${EMACS} -Q \
 --eval "(setq python-environment--verbose t)" \
---eval "(setq jedi:environment-root \"$(PWD)/$(ENV)\")"
+--eval "(setq jedi:environment-root \"$(ENV)\")"
 
 .PHONY : test test-1 tryout clean-elpa requirements env clean-env clean \
 	print-deps travis-ci doc
@@ -72,9 +72,11 @@ install-jedi-dev:
 	${PIP_INSTALL} --upgrade ${JEDI_DEV_URL}
 
 env: $(ENV)/$(BINDIR)/jediepcserver
-$(ENV)/$(BINDIR)/jediepcserver: ${ELPA_DIR} jediepcserver.py setup.py
-	${VIRTUAL_EMACS} -batch -L . -l jedi.el -f "jedi:install-server-block"
-	test -f $@
+$(ENV)/$(BINDIR)/jediepcserver: setup.py
+	${VIRTUAL_EMACS} -batch -L . -l jedi.el \
+		--eval "(setq jedi:install-server--command '(\"pip\" \"install\" \"--editable\" \"$(PWD)\"))" \
+		-f "jedi:install-server-block"
+	test -x $@
 
 clean-env:
 	rm -rf $(ENV)

